android-sdk/src/test/java/org/sagebionetworks/bridge/android/manager/AuthenticationManagerTest.java

```diff
@@ -426,7 +415,6 @@ public void getUserSessionInfo_NullProvider() {
         when(accountDAO.getUserSessionInfo()).thenReturn(previousSession);
 
         SignIn signIn = new SignIn();
-        when(accountDAO.getSignIn()).thenReturn(signIn);
 //        when(apiClientProvider.getUserSessionInfoProvider(signIn)).thenReturn(null);
```

Comment:
How come all these unit tests are still commented out? I believe we only allowed them to be commented out for a QA build months ago